src/backend/distributed/executor/multi_client_executor.c

```diff
@@ -854,23 +857,6 @@ MultiClientWait(WaitInfo *waitInfo)
 
 
 /*
- * ClearRemainingResults reads result objects from the connection until we get
- * null, and clears these results. This is the last step in completing an async
- * query.
- */
-static void
-ClearRemainingResults(MultiConnection *connection)
```

Comment:
To Ozgun's point, this was probably a better name than `ForgetResults`. Would be nice to use this name instead, but can skip over that for now.